pkg/start/start.go

```diff
@@ -72,10 +72,17 @@ func Start(ctx context.Context, inst *store.Instance) error {
 	}
 	// no defer haStderrW.Close()
 
+	var debugFlags []string
+	if logrus.GetLevel() >= logrus.DebugLevel {
+		debugFlags = append(debugFlags, "--debug")
+	}
+
 	haCmd := exec.CommandContext(ctx, self,
-		"hostagent",
-		"--pidfile", haPIDPath,
-		inst.Name)
+		append(debugFlags, []string{
+			"hostagent",
+			"--pidfile", haPIDPath,
+			inst.Name}...)...)
```

Comment:
I find the `}...)...)` construct not very readable. Wouldn't it be better to just build up an `args` list upfront and then call:

```
	var args []string
	if logrus.GetLevel() >= logrus.DebugLevel {
		args = append(args, "--debug")
	}
	args = append(args,
		"hostagent",
		"--pidfile", haPIDPath,
		inst.Name)
	haCmd := exec.CommandContext(ctx, self, args...)
```

Reply:
fixed